packages/gatsby-transformer-pdf/src/gatsby-node.js

```diff
@@ -0,0 +1,49 @@
+const Promise = require(`bluebird`)
+const PDFParser = require(`pdf2json`)
+
+const convertToJson = path =>
+  new Promise((res, rej) => {
+    const pdfParser = new PDFParser(this, 1)
```

Comment:
What's up with the `this, 1` args?

Reply:
the this arg is for something that the plugin [don't need](https://github.com/modesty/pdf2json/blob/master/pdfparser.js#L99) and it should be null, and 1 enables to extract the raw Text.